esp-hal/MIGRATING-0.22.md

```diff
@@ -458,3 +470,6 @@ The Address and Command enums have similarly had their variants changed from e.g
 - Command::Command1
 + Command::_1Bit
 ```
+
+The `write_byte` and `read_byte` was removed. 
+`driver.write_bytes(&[data])?` could be used.
```

Comment:
I would either expand this with something like, you can use `write_bytes`/`read_bytes` as replacement or something like that, but saying that we "could use" `write_bytes` feels a bit short imho